src/taipy/gui/gui.pyi

```diff
@@ -219,6 +220,7 @@ class Gui:
     def _hold_actions(self, callback: t.Optional[t.Union[str, t.Callable]] = ..., message: t.Optional[str] = ...): ...
     def _resume_actions(self) -> None: ...
     def _navigate(self, to: t.Optional[str] = ..., tab: t.Optional[str] = ..., force: t.Optional[bool] = ...): ...
+    def __init_libs(self) -> None: ...
```

Comment:
Do we really want the `_` functions and methods in the pyi file?

Reply:
I don't think so but @dinhlongviolin1 knows better ;-)

Reply:
Lets keep it for now, will come up with new process later (probably this week)